packages/editor/src/plugins/image/components/pixabay-search-button.tsx

```diff
@@ -0,0 +1,256 @@
+
+  const handleKeyDown = async (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key !== 'Enter') return
+    await handleSearch(query)
+  }
+
+  const isGermanLocale = lang === 'de'
+  const showTags = isGermanLocale && images.length === 0 && !isSearching
+
+  return (
+    <>
+      <button
+        data-qa="plugin-image-pixabay-search-button"
+        onClick={() => setIsOpen(true)}
+        onFocus={() => onFocus()}
+        onBlur={() => onBlur()}
+        className="mb-4 flex min-w-full flex-shrink-0 items-center justify-center rounded-lg bg-editor-primary-200 p-1 py-2 font-semibold text-almost-black text-gray-800 hover:bg-editor-primary-300"
+      >
+        <span className="mr-2 inline-block">
+          <FaIcon icon={faMagnifyingGlass} />
+        </span>
+        {imageStrings.searchOnline}
+      </button>
+      <ModalWithCloseButton
+        isOpen={isOpen}
+        setIsOpen={setIsOpen}
+        className="max-h-[700px] w-[900px] max-w-[90vw] pt-0"
+      >
+        <div
+          className={cn(
+            'max-h-[60vw] pt-2',
+            isGermanLocale ? 'min-h-[20vw]' : 'min-h-[5vw]'
+          )}
+        >
+          <h2 className="mb-6 ml-10 mt-10 font-bold">
+            {imageStrings.licenceFree}
+          </h2>
+          {/* Search input */}
+          <div className="relative ml-10 w-[90%]">
+            <input
+              ref={inputRef}
+              type="text"
+              value={query}
+              onChange={handleInputChange}
+              placeholder={imageStrings.search}
+              onKeyDown={handleKeyDown}
+              className="w-full rounded-lg border-0 bg-yellow-100 py-2 pl-4 pr-10 text-gray-600"
+            />
+            {query && (
+              <div
+                className="absolute bottom-0 right-0 top-0 flex cursor-pointer items-center justify-center px-4 text-gray-400"
+                onClick={() => {
+                  setQuery('')
+                  setImages([])
+                  setHasSearched(false)
+                  setTimeout(() => {
+                    inputRef.current?.focus()
+                  })
+                }}
+              >
+                <FaIcon icon={faXmarkCircle} />
+              </div>
+            )}
+          </div>
+
+          {/* Search tags */}
+          {showTags && (
+            <div>
+              <div className="mb-6 mt-10 flex flex-wrap justify-center">
+                {Object.values(germanSearchTags).map((tagKey) => (
+                  <button
+                    key={tagKey}
+                    onClick={() => handleTagClick(tagKey)}
+                    className="m-2 inline-block rounded-md bg-white px-2 py-1 text-sm text-black shadow-md"
+                  >
+                    {tagKey}
+                  </button>
+                ))}
+              </div>
+            </div>
+          )}
+
+          <div
+            className={cn(
+              'max-h-[500px]',
+              'mt-4 flex flex-wrap px-8',
```

Comment:
```suggestion
              'mt-4 flex flex-wrap pl-8 pr-10',
```
(makes the right side of the images line up with the search input